polygon/stocks/stocks.py

```diff
@@ -358,7 +358,7 @@ def get_daily_open_close(self, symbol: str, date, adjusted: bool = True,
     def get_aggregate_bars(self, symbol: str, from_date, to_date, adjusted: bool = True,
                            sort='asc', limit: int = 5000, multiplier: int = 1, timespan='day', full_range: bool = False,
                            run_parallel: bool = True, max_concurrent_workers: int = cpu_count() * 5,
-                           warnings: bool = True, high_volatility: bool = False, raw_response: bool = False):
+                           warnings: bool = True, info: bool = True, high_volatility: bool = False, raw_response: bool = False):
```

Comment:
the only small concern here is that this base method is used by other clients as well. While it won't result in any errors (due to using named default attribute), it changes the base method signature which isn't a good practice.

We can either 
- make this top level change for the other 3 clients (forex, crypto, options)
- OR skip this in this PR. I'll make the change in the next release

Reply:
Makes sense to sync the other clients with the changed API. I'll update the PR in the next couple of days